### ISSUE TITLE
Plantar pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11177]
**Preferred term label:**
Plantar pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0240777
Tenderness to palpation of plantar aspect of the foot

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fifth metatarsal fracture
Calcaneus fractures
Plantar fasciitis
Flexor hallicus longus (FHL) tendon dysfunction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001399
- label: Plantar pain
- parents: [Foot pain (HP:0025238)](https://hpo.jax.org/app/browse/term/HP:0025238)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to palpation of plantar aspect of the foot (the sole).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11177
- synonyms:
